packages/odata-v2/src/request-builder/batch-request-builder.spec.ts

```diff
@@ -92,6 +92,66 @@ HTTP/1.1 200 OK
     }
   });
 
+  it('batch works with content id', async () => {
+    const body = [
+      '--batch_test-boundary',
+      'Content-Type: multipart/mixed; boundary=changeset_test-boundary',
+      '',
+      '--changeset_test-boundary',
+      'Content-Type: application/http',
+      'Content-Transfer-Encoding: binary',
+      'Content-Id: myContentId',
+      '',
+      'POST /sap/opu/odata/sap/API_TEST_SRV/A_TestEntity HTTP/1.1',
+      'Content-Type: application/json',
+      'Accept: application/json',
+      '',
+      '{}',
+      '',
+      '--changeset_test-boundary',
+      'Content-Type: application/http',
+      'Content-Transfer-Encoding: binary',
+      'Content-Id: test-boundary',
+      '',
+      "POST $myContentId/TestFunctionImportPOST?SimpleParam='someValue' HTTP/1.1",
```

Comment:
As discussed, this example is an unsupported odata feature, bound function.
Maybe add more examples by e.g., searching "content-id" [here](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html)

Reply:
sure I adjust the example. I ping you once the draft is updated.